### DISCUSSION TITLE
Conversion of an A4 to A5 page with half a blank page

### DISCUSSION THREAD

**Discussion by MisiekDP** [ImageMagick/ImageMagick#5758] [Help] [answered]
Hey,
I need to conversion of an A4 to A5 page with half a blank page. I hava this command: `convert -density 600 INPUT.pdf -rotate 90 -colorspace sRGB -background white -alpha background -alpha remove +level-colors magenta,white -quality 100 OUT.pdf`.

Generally I have this file:
![yl29D](https://user-images.githubusercontent.com/22171619/202265328-75b1293d-9904-4ffb-a979-0f0babb3cc71.png)

And I need this:
![2PAOe2](https://user-images.githubusercontent.com/22171619/202265357-a741358d-7b0e-44d1-90e0-47c4613b66e6.png)

I don't know, maybe it's a good idea to add a white border 50% of the page width at the top of the page (the page is rotated 90 degrees, so it's horizontal). And it could be good to rotate and transform vectors first and after this rasterize do output pdf file.

**Comment by snibgo** [comment, marked as answer]
When IM reads a PDF, it makes a raster image. Your processing modifies that image. You don't have any processing to create the blank bottom half of the image. I suggest this:
```
( +clone -fill White -colorize 100 ) -append +repage
```
This clones the image, makes that white, and appends it below its input. So the complete command would be:
```
convert -density 600 INPUT.pdf -rotate 90 -colorspace sRGB -background white -alpha background -alpha remove +level-colors magenta,white -quality 100 ( +clone -fill White -colorize 100 ) -append +repage OUT.pdf
```

**Reply by MisiekDP** [reply]
It works awesome! :D Love ImageMagick!
